integration-tests/field-validation/src/index.exo

```diff
@@ -5,5 +5,7 @@ module ExampleDatabase {
     @pk id: Int = autoIncrement()
     @range(min = 0, max = 2) limit: Int?
     age: Int?
+    @range(min = "-1.0", max = "3.0") flimit: Float?
```

Comment:
notice quotes (`min` and `max` values are strings. Otherwise integration tests fail with
```
error[S000]: Unexpected token: "."
 --> src/index.exo:8:19
  |
8 |     @range(min = 0.0, max = 2.0) flimit: Float?
  |                   ^ unexpected "."
error[S000]: Unexpected token: "."
 --> src/index.exo:8:30
  |
8 |     @range(min = 0.0, max = 2.0) flimit: Float?
  |                              ^ unexpected "."

Error: Parser error: Could not process input exo files
```

Reply:
Let's stay with strings for now. We can separately deal with the parser issue.